examples/opensora_pku/opensora/models/diffusion/latte/modeling_latte.py

```diff
@@ -454,6 +468,18 @@ def construct(
         # b d -> (b p) d
         timestep_temp = timestep.repeat_interleave(num_patches, dim=0)
 
+        # BS H -> S B H
+        if get_sequence_parallel_state():
+            timestep_temp = timestep_temp.view(input_batch_size * num_patches, 6, -1).swapaxes(0, 1).contiguous()
+            # f, h -> f, 1, h
+            temp_pos_embed = self.temp_pos_embed[self.temp_pos_st : self.temp_pos_ed].unsqueeze(1)
+        else:
+            temp_pos_embed = self.temp_pos_embed[: self.video_length].unsqueeze(0)
+            if temp_pos_embed.shape[1] != frame:
```

Comment:
In which case that `frame` is greater than `temp_pos_embed.shape[1]`? If they are different, either the `temp_pos_embed` shape is wrong or the output shape of the vae encoder is wrong. I don't padding here is the correct way to solve this problem.

Reply:
This is the writing method that follows the official implementation, but perhaps an exception should be thrown directly here.
https://github.com/PKU-YuanGroup/Open-Sora-Plan/blob/hw/opensora/models/diffusion/latte/modeling_latte.py#L459